src/bitcoin-cli.cpp

```diff
@@ -481,6 +481,16 @@ class NetinfoRequestHandler : public BaseRequestHandler
         }
         return str;
     }
+    std::string ServicesList(const UniValue& services)
+    {
+        std::vector<std::string> v;
+        for (size_t i = 0; i < services.size(); ++i) {
```

Comment:
should we maybe sort to avoid an order like:
```bash
Local services: network, witness, network limited, p2p v2
```

it may make sense to have:
```bash
Local services: network, network limited, p2p v2, witness
```

Reply:
I kept the order the same as in getpeerinfo RPC, the GUI, and the -netinfo peers list.